schemas/questions.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "question",
+      "description",
+      "type"
+    ],
+    "properties": {
+      "question": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "validation": {
+        "type": "string"
```

Comment:
i would prefere the validation to be a enum with limited options

Reply:
Our intention is to use regex here so a string would be the best type.